src/main/java/net/malisis/core/client/gui/component/interaction/UISelect.java

```diff
@@ -446,7 +446,7 @@ public T select(Option<T> option)
 	 */
 	public T select(T obj)
 	{
-		return select(getOption(obj));
+		return (T) select(getOption(obj));
```

Comment:
idk what you have enabled in Eclipse but this would not compile in IntelliJ nor gradle for me without the cast.